docs/tutorials/web/ep-webnn.md

```diff
@@ -140,25 +140,25 @@ const results = await mySession.run(feeds, fetches);
 
 ```
 
-By specifying the output tensor in the fetches, ONNX Runtime Web will use the pre-allocated buffer as the output buffer. If there is a shape mismatch, the `run()` call will fail.
+By specifying the output tensor in the fetches, ONNX Runtime Web will use the pre-allocated MLTensor as the output tensor. If there is a shape mismatch, the `run()` call will fail.
 
 ### Specify the output data location
 
-If you don't want to use pre-allocated MLBuffer tensors for outputs, you can also specify the output data location in the session options:
+If you don't want to use pre-allocated MLTensor tensors for outputs, you can also specify the output data location in the session options:
 
 ```js
 const mySessionOptions1 = {
   ...,
-  // keep all output data on MLBuffer
-  preferredOutputLocation: 'ml-buffer'
+  // keep all output data on MLTensor
+  preferredOutputLocation: 'ml-tensor'
```

Comment:
~~Is there also a cpp/js file somewhere that needs to be updated to check for this string?~~ Never mind - this is just the doc.